biz.aQute.bnd.util/src/aQute/bnd/unmodifiable/Maps.java

```diff
@@ -62,6 +62,12 @@ private Maps() {}
 			entry(k6, v6), entry(k7, v7), entry(k8, v8), entry(k9, v9), entry(k10, v10));
 	}
 
+	public static <K, V> Map<K, V> of(K k1, V v1, K k2, V v2, K k3, V v3, K k4, V v4, K k5, V v5, K k6, V v6, K k7,
```

Comment:
Lets not do this. Just use the existing methods like ofEntries.

Reply:
DONE